net-im/discord/discord-0.0.84.ebuild

```diff
@@ -143,4 +143,10 @@ pkg_postinst() {
 	optfeature "sound support" \
 		media-sound/pulseaudio media-sound/apulse[sdk] media-video/pipewire
 	optfeature "emoji support" media-fonts/noto-emoji
+	if has_version kde-plasma/kwin[-screencast] && use wayland; then
+		einfo " "
+		einfo "On Wayland you need to enable kde-plasna/kwin[screencast] USE flag"
+		einfo "in order to be able to share your screen if you're using Kwin."
```

Comment:
can we reword it to

```
When using KWin on Wayland, the kde-plasma/kwin[screencast] USE flag must be enabled for screensharing 
```

or something like that, i wanted to remove `you`

Reply:
Absolutely.